### PR TITLE
Uploaded newversion file

### PR DESCRIPTION
**Pull request by ankitdobhal**
* To create newversion command or tool recommended as in https://github.com/sclorg/container-workflow-tool/issues/10 .
* I m implementing gitpython module to develop the kind of tool.

### CODE REVIEW

**Comment by phracek** [comment]
First of all, you have to add here a new command `newversion` https://github.com/sclorg/container-workflow-tool/blob/master/container_workflow_tool/constants.py#L38

```python
action_map['newversion'] = {
'add': 'add_new_version',
'show': 'show_changes',
}
```

And here create a new python file called `newversion.py` https://github.com/sclorg/container-workflow-tool/tree/master/container_workflow_tool

The `newversion.py` should be similar to:
```python

import os
import shutil

from git import Repo
from git.exc import GitCommandError

class NewVersion(object):
```

In to the file https://github.com/sclorg/container-workflow-tool/blob/master/container_workflow_tool/cli.py#L28 , you have to add a function like

```python
    def newversion_usage(self):
        return CliCommon.newversion_usage(self) % (self.prg_name, "")
```

and the `newversion_usage` itself will be defined here:
https://github.com/sclorg/container-workflow-tool/blob/master/container_workflow_tool/cli_common.py#L88

Similar to git_usage.

File: container_workflow_tool/newversion.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import os
+import subprocess as sub
+import shutil
+
+from git import Repo
+from git.exc import GitCommandError
+
+class NewVersion(object,):
```
[the remark this answers is not in the document]
Comment (ankitdobhal) [reply container_workflow_tool/newversion.py]: Resolved This issue in new commit #11

File: container_workflow_tool/newversion.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	def __init__(self,_oldversion,_version):
+		self._oldversion=_oldversion
+		self._version=_version
+
+		def create_newversion(_version):
```
[the remark this answers is not in the document]
Comment (ankitdobhal) [reply container_workflow_tool/newversion.py]: Resolved This issue in new commit #11

File: container_workflow_tool/newversion.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ankitdobhal) [reply container_workflow_tool/newversion.py]: >>
lets say we have 1.9 as oldversion so the script is working to update it into the newversion as 2.0.
Screenshot from 2020-05-08 18-09-30

File: container_workflow_tool/constants.py [hunk 3]
```diff
@@ -36,4 +36,9 @@
 actions['dockerhub'] = ['updatefulldescription', ]
 actions['utils'] = ['showconfig', 'listimages', 'listupstream', ]
```
[the remark this answers is not in the document]
Comment (ankitdobhal) [reply container_workflow_tool/constants.py:38]: But I need some instructions to add commands for newversion.
for #11

[the remark this answers is not in the document]
Comment (phracek) [reply container_workflow_tool/constants.py:38]: Take inspiration from the other `cwt` subcommands like `git`. This is a part of your job ;)

**Comment by phracek** [comment]
Closing this pull request. We will not implement it for now.

**Pull request closed without merging**